packages/core/src/Stores/traders-hub-store.js

```diff
@@ -153,21 +152,21 @@ export default class TradersHubStore extends BaseStore {
             }
         );
 
-        reaction(
-            () => [
-                this.root_store.client.balance,
-                this.root_store.client.loginid,
-                this.root_store.client.obj_total_balance,
-                this.root_store.client.mt5_login_list,
-                this.root_store.client.dxtrade_accounts_list,
-                this.root_store.accounts,
-                this.selected_account_type,
-                this.selected_region,
-            ],
-            async () => {
-                await this.updatePlatformBalance();
-            }
-        );
+        // reaction(
```

Comment:
@shayan-deriv Please remove the commented code 🙏🏻

Reply:
I need to check these dependencies to make sure which one of them needs to move to my useMemo dependency list. will remove it once my check is done